### DISCUSSION TITLE
how to change the length of a video

### DISCUSSION THREAD

**Discussion by Filib91** [nerdyrodent/VQGAN-CLIP#133] [Q&A]
hello,
im using this command to generate a video but its to short ... i want a 20sec (30fps) if possible. Please help me with these settings

**Comment by mgatelabs** [comment]
I was playing with that last night with this call

`python generate.py -p "You and I are pilgrims of the galaxy" -zvid -zse 50 -vl 10 -zsc 0.97 -ofps 30 -opt Adam -lr 0.15 -s 256 256 -ckpt checkpoints/coco.ckpt -conf checkpoints/coco.yaml`

But it didn't really make anything, just like 1 second.

So what I did was say, `-zse 50` so every 50 iterations is one frame, so 30 seconds at 30 fps is 30 * 30 = 900, but you need iterations so 900 * 50 = 45000.  So set -i tp 45000 and -vl 30.

Still not sure though, the whole video thing is weird, think I need to pull in that patch for the timeline tool, it looked sick.
